packages/nextjs/app/builders/_components/Batch4Avatar.tsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from "react";
+import { AvatarComponent } from "@rainbow-me/rainbowkit";
+import { blo } from "blo";
+import { getAddress, isAddress } from "viem";
+import { useEnsAvatar, useEnsName } from "wagmi";
+
+export const Batch4Avatar: AvatarComponent = ({ address, size }) => {
+  const [ensAvatar, setEnsAvatar] = useState<string | null>();
+  const checkSumAddress = address ? getAddress(address) : undefined;
+
+  const { data: fetchedEns } = useEnsName({
+    address: checkSumAddress,
+    enabled: isAddress(checkSumAddress ?? ""),
+    chainId: 1,
+  });
+
+  const { data: fetchedEnsAvatar } = useEnsAvatar({
+    name: fetchedEns,
+    enabled: Boolean(fetchedEns),
+    chainId: 1,
+    cacheTime: 30_000,
+  });
+
+  useEffect(() => {
+    setEnsAvatar(fetchedEnsAvatar);
+  }, [fetchedEnsAvatar]);
+
+  return (
+    <img
```

Comment:
This is giving this linting error. You can ignore it with: // eslint-disable-next-line @next/next/no-img-element

![image](https://github.com/BuidlGuidl/batch4.buidlguidl.com/assets/2486142/178ade2a-de2b-48ac-80f7-ce617c7994d8)